src/main/rx/cc2500_sfhss.c

```diff
@@ -0,0 +1,448 @@
+                initialise();
+                SET_STATE(STATE_BIND);
+                DEBUG_SET(DEBUG_RX_SFHSS_SPI, DEBUG_DATA_MISSING_FRAME, dataMissingFrame);
+            }
+            break;
+        case STATE_BIND:
+            if (cc2500checkBindRequested(true)) {
+                cc2500LedOn();
+                initTuneRx();
+                SET_STATE( STATE_BIND_TUNING1 );
+            } else {
+                SET_STATE( STATE_HUNT );
+                sfhssnextChannel();
+                setRssiDirect(0, RSSI_SOURCE_RX_PROTOCOL);
+                nextFrameReceiveStartTime = currentPacketReceivedTime + NEXT_CH_TIME_HUNT;
+            }
+            break;
+        case STATE_BIND_TUNING1:
+           if (tune1Rx(packet)) {
+                SET_STATE( STATE_BIND_TUNING2 );
+            }
+            break;
+        case STATE_BIND_TUNING2:
+           if (tune2Rx(packet)) {
+                SET_STATE( STATE_BIND_TUNING3);
+            }
+            break;
+        case STATE_BIND_TUNING3:
+           if (tune3Rx(packet)) {
+                if(((int16_t)bindOffset_max - (int16_t)bindOffset_min) <= 2){
+                    initTuneRx();
+                    SET_STATE( STATE_BIND_TUNING1 );    // retry
+                }else{
+                    rxSfhssSpiConfigMutable()->bindOffset = ((int16_t)bindOffset_max + (int16_t)bindOffset_min)/2 ;
+                    SET_STATE(STATE_BIND_COMPLETE);
+                }
+            }
+            break;
+        case STATE_BIND_COMPLETE:
+            writeEEPROM();
+            ret = RX_SPI_RECEIVED_BIND;
+            SET_STATE(STATE_INIT);
+            break;
+        case STATE_HUNT:
+            if (sfhssRecv(packet)){
+                if (sfhssPacketParse(packet, true)){
+                    if (GET_COMMAND(packet) & 0x8){       /* ch=5-8 */
+                        missingPackets = 0;
+                        cc2500LedOn();
+                        frame_recvd = 0x3;
+                        SET_STATE(STATE_SYNC);
+                        nextFrameReceiveStartTime = currentPacketReceivedTime + NEXT_CH_TIME_SYNC2;
+                        return RX_SPI_RECEIVED_NONE;
+                    }
+                }
+                cc2500Strobe(CC2500_SRX);
+            }else if (cmpTimeUs(currentPacketReceivedTime, nextFrameReceiveStartTime) > 0) {
+                cc2500LedBlink(500);
+#if defined(USE_RX_CC2500_SPI_PA_LNA) && defined(USE_RX_CC2500_SPI_DIVERSITY) // SE4311 chip
+                cc2500switchAntennae();
+#endif
+                sfhssnextChannel();
+                nextFrameReceiveStartTime += NEXT_CH_TIME_HUNT;
+            }else if (cc2500checkBindRequested(false)) {
+                SET_STATE(STATE_INIT);
+                break;
+            }
+            break;
+        case STATE_SYNC:
+            if (sfhssRecv(packet)){
+                if (sfhssPacketParse(packet, true)){
+                    missingPackets = 0;
+                    if( GET_COMMAND(packet) & 0x8 ){ 
```

Comment:
Trailing space at the end of the line `& 0x8 ){`.

Reply:
fixed